### PR TITLE
Update Cura documentation and instructions

### PR DESCRIPTION
**Pull request by HumbleDeer**
## Changes in this PR:

- Replaced Octoprint plugin instructions with instructions for installing and using the newer Moonraker Connection plugin
- Added extra clarification and steps regarding the configuration of the Moonraker Connection plugin. 
- Replaced an outdated image with a new one, which reflects the interface that the Moonraker Connection plugin for Cura provides by default.

## Merging

I suggest using Squash Merge for this pull request, and condensing this list of changes into a short and concise version.

```text
Update Cura for Moonraker Connection plugin

Updated the Cura instructions for the Moonraker Connection plugin, which replaces the old OctoPrint Connection plugin.
```

### CODE REVIEW

File: overview/slicer/cura.md [hunk 1]
```diff
@@ -2,25 +2,42 @@
...
+    The address you've been using to access Mainsail, prefixed by `http://`.  
+    e.g. `http://myprinter.local/`
+  - Monitor > **Camera > URL**
+    The address of the camera stream on the network, prefixed by `http://`.
+    e.g. `http://myprinter.local/webcam/stream`
 
-<figure><img src="../../.gitbook/assets/image (26).png" alt=""><figcaption></figcaption></figure>
+<figure><img src="../../.gitbook/assets/cura_moonraker-connection_configuration-window.jpg" alt="Moonraker Connection configuration window in Cura"><figcaption>Moonraker Connection configuration window in Cura</figcaption></figure>
 
 ### Best practise G-codes
```
Comment (HumbleDeer) [inline overview/slicer/cura.md:38]:
```suggestion
## Best practise G-codes
```

File: overview/slicer/cura.md [hunk 2]
```diff
@@ -2,25 +2,42 @@
...
+You have to add a Post Processing Script to add "G-code thumbnails" in Cura. To add these scripts, click on `Extensions > Post Processing > Modify Gcode`. Then click on `Add a script`, choose `Create Thumbnail` and fill in the size `32x32`. Add again the same script with the size `400x400`.
 
 {% hint style="warning" %}
-Important is, that the size is square! If you don't use a square resolution, Cura will squeeze the thumbnail.
+It is important for this size to be a square, with equal size on both length and width. If you don't use a square resolution, Cura will squeeze the thumbnail.
 {% endhint %}
 
 <figure><img src="../../.gitbook/assets/image (36).png" alt=""><figcaption></figcaption></figure>
 
 ### Direct G-code upload
```
Comment (HumbleDeer) [inline overview/slicer/cura.md:19]:
```suggestion
## Direct G-code upload
```

File: overview/slicer/cura.md [hunk 3]
```diff
@@ -2,25 +2,42 @@
 
 ### Thumbnail
```
Comment (HumbleDeer) [inline overview/slicer/cura.md:3]:
```suggestion
## Thumbnail
```

**Review by HumbleDeer — COMMENTED** [review]
Optional change to merge alongside or separately:
Changes the level 3 headings to level 2 headings, in accordance and respect of the general Markdown syntax rules and guidelines.

**Review by meteyou — APPROVED** [review]

**Comment by meteyou** [comment]
@HumbleDeer Thank you very much for your PR!

I use Cura very little, but do you think we can replace the whole guide with the script from Pedro and remove the rest of the instructions?

**Comment by HumbleDeer** [comment]
I'd argue the reverse would be more useful. The moonraker connection plugin
does previews too. The script would hence not need to create them anymore.

The plugin does more than just previews. It provides means to send files to
the server, and more.

I'm intent on updating the plugin itself to allow more features that the
octoprint one supports too, such as temperature previews and controls
etcetera. That'll be somewhere in the far future.

Thanks,
HumbleDeer

**Pull request merged**